### PR TITLE
Bug/skale 3891 refill less

### PR DESCRIPTION
**Pull request by yavrsky**

### CODE REVIEW

**Review by payvint — DISMISSED** [review]
Good

**Review by DimaStebaev — DISMISSED** [review]

File: contracts/SkaleDKG.sol [hunk 1]
```diff
@@ -180,22 +188,27 @@ contract SkaleDKG is Permissions, ISkaleDKG {
...
         _refundGasByValidatorToSchain(schainId);
     }
 
     function alright(bytes32 schainId, uint fromNodeIndex)
         external
-        refundGasBySchain(schainId, false)
+        refundGasBySchain(schainId, 
+            Context({
+                isDebt: false,
+                delta: ConstantsHolder(contractManager.getContract("ConstantsHolder")).ALRIGHT_DELTA(),
```
Comment (payvint) [inline contracts/SkaleDKG.sol]: Maybe let's change it to getConstantsHolder - instead of getContract

Comment (yavrsky) [reply contracts/SkaleDKG.sol]: ok

**Review by payvint — DISMISSED** [review]
Looks good

**Review by DimaStebaev — APPROVED** [review]

**Review by payvint — APPROVED** [review]

**Pull request merged**